bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/TimeAmount.scala

```diff
@@ -13,7 +16,9 @@ object TimeAmount {
   // with the difference that we try to keep the time unit from changing if they
   // are all the same.
   def sum(times: Traversable[TimeAmount]): TimeAmount =
-    if (times.isEmpty) empty else (times.head/:times.tail)(_ + _)
+    if (times.isEmpty) empty else (times.head/:times.tail)(_ |+| _)
```

Comment:
Yeah, I was thinking that. In `Submissions.scala`, though, we have:

`  def sumTimes(subs: Traversable[Submission], f: Submission => TimeAmount): TimeAmount =
    subs.map(f).suml`

and even with `import scalaz._, Scalaz._`, I'm getting:

Error:(32, 17) value suml is not a member of Traversable[edu.gemini.model.p1.immutable.TimeAmount]
subs.map(f).suml